src/ReactWrapper.js

```diff
@@ -286,6 +286,17 @@ export default class ReactWrapper {
   }
 
   /**
+   * Returns the HTML of the node.
+   *
+   * NOTE: can only be called on a wrapper of a single node.
+   *
+   * @returns {String}
+   */
+  html() {
+    return this.single(n => findDOMNode(n).outerHTML.replace(/\sdata-reactid[^>]+/g, ''));
```

Comment:
how safe is this? if I understand correctly, this is assuming that the `data-reactid` attribute will be the last attribute of the node...  how sure are we that this will be the case?  A more robust regex might be good...  or perhaps another approach to get this html?